integration-cli/docker_cli_swarm_test.go

```diff
@@ -52,12 +52,20 @@ func (s *DockerSwarmSuite) TestSwarmUpdate(c *check.C) {
 	spec = getSpec()
 	c.Assert(spec.CAConfig.NodeCertExpiry, checker.Equals, 30*time.Hour)
 
+	tempFile, err := ioutil.TempFile("", "testfile")
+	c.Assert(err, checker.IsNil)
+	defer os.Remove(tempFile.Name())
+	tempFile.Write([]byte("fakecert"))
+
 	// passing an external CA (this is without starting a root rotation) does not fail
-	out, err = d.Cmd("swarm", "update", "--external-ca", "protocol=cfssl,url=https://something.org")
+	out, err = d.Cmd("swarm", "update", "--external-ca", "protocol=cfssl,url=https://something.org",
+		"--external-ca", fmt.Sprintf("protocol=cfssl,url=https://somethingelse.org,cacert=%s", tempFile.Name()))
```

Comment:
Using this will give much better error messages when there is a failure:

```go
result := cli.Docker(cli.Args(...), cli.Daemon(d.Daemon))
result.Assert(c, icmd.Success)
```